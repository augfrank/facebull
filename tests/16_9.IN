M0   C0   C1  2690
M1   C0   C2  2770
M2   C0   C3  2603
M3   C0   C4  2080
M4   C0   C5  2344
M5   C0   C6  2734
M6   C0   C7  2327
M7   C0   C8  2953
M8   C0   C9  2937
M9   C0   C10  2449
M10   C0   C11  2900
M11   C0   C12  2965
M12   C0   C13  2064
M13   C0   C14  2846
M14   C0   C15  2513
M15   C1   C0  2623
M16   C1   C2  2346
M17   C1   C3  2917
M18   C1   C4  2090
M19   C1   C5  2206
M20   C1   C6  2127
M21   C1   C7  2635
M22   C1   C8  2393
M23   C1   C9  2674
M24   C1   C10  2432
M25   C1   C11  2964
M26   C1   C12  2373
M27   C1   C13  2174
M28   C1   C14  2431
M29   C1   C15  2398
M30   C2   C0  2314
M31   C2   C1  2899
M32   C2   C3  2695
M33   C2   C4  2788
M34   C2   C5  2639
M35   C2   C6  2179
M36   C2   C7  2076
M37   C2   C8  2401
M38   C2   C9  2272
M39   C2   C10  2130
M40   C2   C11  2653
M41   C2   C12  2793
M42   C2   C13  2427
M43   C2   C14  2422
M44   C2   C15  2834
M45   C3   C0  2577
M46   C3   C1  2430
M47   C3   C2  2660
M48   C3   C4  2169
M49   C3   C5  2369
M50   C3   C6  2162
M51   C3   C7  2689
M52   C3   C8  2536
M53   C3   C9  2639
M54   C3   C10  2961
M55   C3   C11  2673
M56   C3   C12  2496
M57   C3   C13  2486
M58   C3   C14  2919
M59   C3   C15  2118
M60   C4   C0  2509
M61   C4   C1  2960
M62   C4   C2  2204
M63   C4   C3  2916
M64   C4   C5  2446
M65   C4   C6  2706
M66   C4   C7  2280
M67   C4   C8  2801
M68   C4   C9  2552
M69   C4   C10  2838
M70   C4   C11  2589
M71   C4   C12  2335
M72   C4   C13  2297
M73   C4   C14  2064
M74   C4   C15  2126
M75   C5   C0  2084
M76   C5   C1  2259
M77   C5   C2  2006
M78   C5   C3  2499
M79   C5   C4  2698
M80   C5   C6  2812
M81   C5   C7  2874
M82   C5   C8  2699
M83   C5   C9  2930
M84   C5   C10  2768
M85   C5   C11  2896
M86   C5   C12  2898
M87   C5   C13  2122
M88   C5   C14  2460
M89   C5   C15  2516
M90   C6   C0  2525
M91   C6   C1  2569
M92   C6   C2  2442
M93   C6   C3  2736
M94   C6   C4  2394
M95   C6   C5  2263
M96   C6   C7  2557
M97   C6   C8  2394
M98   C6   C9  2653
M99   C6   C10  2556
M100   C6   C11  2202
M101   C6   C12  2998
M102   C6   C13  2866
M103   C6   C14  2288
M104   C6   C15  2837
M105   C7   C0  2736
M106   C7   C1  2568
M107   C7   C2  2301
M108   C7   C3  2271
M109   C7   C4  2379
M110   C7   C5  2161
M111   C7   C6  2646
M112   C7   C8  2669
M113   C7   C9  2533
M114   C7   C10  2182
M115   C7   C11  2946
M116   C7   C12  2740
M117   C7   C13  2352
M118   C7   C14  2774
M119   C7   C15  2615
M120   C8   C0  2707
M121   C8   C1  2411
M122   C8   C2  2190
M123   C8   C3  2244
M124   C8   C4  2417
M125   C8   C5  2710
M126   C8   C6  2317
M127   C8   C7  2003
M128   C8   C9  2857
M129   C8   C10  2732
M130   C8   C11  2353
M131   C8   C12  2019
M132   C8   C13  2298
M133   C8   C14  2050
M134   C8   C15  2160
M135   C9   C0  2070
M136   C9   C1  2645
M137   C9   C2  2846
M138   C9   C3  2807
M139   C9   C4  2491
M140   C9   C5  2323
M141   C9   C6  2742
M142   C9   C7  2837
M143   C9   C8  2926
M144   C9   C10  2334
M145   C9   C11  2306
M146   C9   C12  2680
M147   C9   C13  2918
M148   C9   C14  2643
M149   C9   C15  2042
M150   C10   C0  2423
M151   C10   C1  2114
M152   C10   C2  2127
M153   C10   C3  2391
M154   C10   C4  2129
M155   C10   C5  2582
M156   C10   C6  2884
M157   C10   C7  2261
M158   C10   C8  2296
M159   C10   C9  2774
M160   C10   C11  2953
M161   C10   C12  2472
M162   C10   C13  2555
M163   C10   C14  2503
M164   C10   C15  2093
M165   C11   C0  2842
M166   C11   C1  2675
M167   C11   C2  2628
M168   C11   C3  2921
M169   C11   C4  2725
M170   C11   C5  2010
M171   C11   C6  2363
M172   C11   C7  2287
M173   C11   C8  2667
M174   C11   C9  2209
M175   C11   C10  2955
M176   C11   C12  2164
M177   C11   C13  2066
M178   C11   C14  2622
M179   C11   C15  2817
M180   C12   C0  2906
M181   C12   C1  2505
M182   C12   C2  2230
M183   C12   C3  2904
M184   C12   C4  2251
M185   C12   C5  2827
M186   C12   C6  2168
M187   C12   C7  2246
M188   C12   C8  2745
M189   C12   C9  2341
M190   C12   C10  2098
M191   C12   C11  2078
M192   C12   C13  2659
M193   C12   C14  2642
M194   C12   C15  2954
M195   C13   C0  2255
M196   C13   C1  2058
M197   C13   C2  2367
M198   C13   C3  2213
M199   C13   C4  2752
M200   C13   C5  2923
M201   C13   C6  2449
M202   C13   C7  2438
M203   C13   C8  2089
M204   C13   C9  2673
M205   C13   C10  2545
M206   C13   C11  2714
M207   C13   C12  2797
M208   C13   C14  2842
M209   C13   C15  2299
M210   C14   C0  2330
M211   C14   C1  2092
M212   C14   C2  2063
M213   C14   C3  2798
M214   C14   C4  2381
M215   C14   C5  2142
M216   C14   C6  2631
M217   C14   C7  2757
M218   C14   C8  2271
M219   C14   C9  2004
M220   C14   C10  2272
M221   C14   C11  2220
M222   C14   C12  2473
M223   C14   C13  2082
M224   C14   C15  2701
M225   C15   C0  2613
M226   C15   C1  2172
M227   C15   C2  2222
M228   C15   C3  2183
M229   C15   C4  2919
M230   C15   C5  2319
M231   C15   C6  2417
M232   C15   C7  2886
M233   C15   C8  2162
M234   C15   C9  2535
M235   C15   C10  2158
M236   C15   C11  2615
M237   C15   C12  2678
M238   C15   C13  2705
M239   C15   C14  2485
