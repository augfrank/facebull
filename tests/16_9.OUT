33478
12 20 36 59 73 77 100 108 131 135 154 170 190 196 219 233
